bitnami/harbor/templates/jobservice/jobservice-svc.yaml

```diff
@@ -12,8 +13,8 @@ metadata:
   {{- end }}
 spec:
   ports:
-    - name: http
-      port: {{ include "harbor.jobservice.servicePort" . }}
-      targetPort: http
+    - name: {{ ternary "https" "http" .Values.internalTLS.enabled }}
+      port: {{ ternary 443 80 .Values.internalTLS.enabled }}
```

Comment:
Same concern here about not customizing the port number

Reply:
To be resolved in previous conversation.